### DISCUSSION TITLE
Welcome to amperfy Discussions!

### DISCUSSION THREAD

**Discussion by BLeeEZ** [BLeeEZ/amperfy#90] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by f-longobardi** [comment]
Hi!

Just found out about this project, I'm using the app on my phone to connect Navidrome server and stream music with CarPlay and since I'm an iOS dev I'll gladly help a bit when I can.

**Comment by jacobdanielrose** [comment]
I really love this app! One thing I'd suggest is adding WatchOS support. I'd love to contribute but it would be a bit of a learning curve since I don't really have much experience with iOS or WatchOS development but I might take this opportunity to learn 🙂
